test_util/test_upgrade_vpc.py

```diff
@@ -271,7 +271,8 @@ def test_app_dns_survive_upgrade():
             self.log_test("test_upgrade_vpc.test_app_dns_survive_upgrade", test_app_dns_survive_upgrade)
 
     def run_test(self) -> int:
-        stack_name = 'upgrade-test-' + random_id(10)
+        stack_prefix = os.getenv("CF_STACK_NAME_PREFIX", "dcos-upgrade-test")
```

Comment:
Addressed. I turned it into a SUFFIX. My need was to have a differentiator that can be set from CI. Having it as a suffix was better, because if we have it a prefix and it is set to blank, the string could start with `-`.